modules/openxr/openxr_api.cpp

```diff
@@ -2028,6 +2057,21 @@ OpenXRAPI::OpenXRAPI() {
 				break;
 		}
 
+		int ebm = GLOBAL_GET("xr/openxr/environment_blend_mode");
```

Comment:
Is `blend_mode` used here, otherwise I think it'd be more descriptive

Reply:
Not entirely sure what you mean by that?

This is basically the default mode for the environment blend mode that you can set in project settings so it is applied seeing OpenXR starts before any user code is run.

Reply:
Meant a clearer name over `ebm`

Reply:
Environment in this case has nothing to do with the Godot environment, it's the environment in which the XR headset is used. In most practical situations, the environment referring to the real world around you, ergo how do we blend the camera image together with what is being rendered?

Reply:
Ah, ok no worries, because those temporary variables overlap in name with the actual member variables I shorthanded them. It's just to do the integer to proper enum conversion :)